scripts/main.js

```diff
@@ -113,6 +113,13 @@ module.exports = (robot) => {
             res.reply("https://docs.google.com/spreadsheets/d/"+process.env.GOOGLE_WORKSHEET+"#gid=1977716389")
         }
     })
+    robot.hear(/bot standups/i, (res) => {
+        let message = res.message.text
+        message = message.split(' ')
+        if(message.length === 2) {
+            res.reply("https://docs.google.com/spreadsheets/d/"+process.env.GOOGLE_WORKSHEET+"#gid=1440664551")
```

Comment:
`"#gid=1440664551"` ? don't think this is a good idea, and don't think this is needed at all - it works without it as well https://docs.google.com/spreadsheets/d/1w4LBF6wbRNVynAk8cQURyz8yZCbTw5hPwcqr87S46hY

Reply:
@zelima but, that `gid` is for the specific `sheet`

Reply:
@Mikanebu I understand, but it is hardcoded there. what if we will delete that sheet and recreate? can you somehow get that ID fromgoogle api Eg with the name- there should be way